src/Microsoft.Azure.WebJobs.Host/Config/FluentBindingRule.cs

```diff
@@ -19,6 +17,7 @@ namespace Microsoft.Azure.WebJobs.Host.Config
     /// Helpers for adding binding rules to a given attribute.
     /// </summary>
     /// <typeparam name="TAttribute"></typeparam>
+    [Obsolete("Not ready for public consumption.")]
     public class FluentBindingRule<TAttribute> : FluentConverterRules<TAttribute, FluentBindingRule<TAttribute>>
```

Comment:
These Fluent ones are likely stable.

Reply:
It's base class FluentConverterRules is also public and is Obsolete. Given that, I think it makes sense to be consistent here. We can always remove these in the next release once we're finished with the APIs.